packages/listview/src/components/ColoredStatusColumn/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useTheme from 'react-rainbow-components/libs/hooks/useTheme.js';
+import getContrastText from 'react-rainbow-components/styles/helpers/color/getContrastText.js';
+import { StyledContainer } from './styled';
+
+const ColoredStatusColumn = ({ value, colors, textTransform }) => {
+    const theme = useTheme();
+
+    const { success, warning, error } = theme.rainbow.palette;
+    const defaultColors = {
+        success: success.main,
+        warning: warning.main,
+        error: error.main,
+    };
+
+    const map = {
+        ...defaultColors,
+        ...colors,
```

Comment:
what happens is the colors is undefined and you try to use spread
```
Uncaught ReferenceError: colors is not defined
```

Reply:
also, you are teaching new colors, no completely overriding the colors

Reply:
... and the components shouldn't be case sensitive, `Success` should be the same as `success`